content/applications/productivity/studio/pdf_reports.rst

```diff
@@ -2,341 +2,542 @@
+   attribute to define the condition, which is then evaluated as `True` or `False`. The table is
+   not displayed if there are no tags in the quotation.
+
+   .. code-block:: xml
+      :emphasize-lines: 2
+
+      <!-- table root element -->
+      <table class="table" t-if="len(doc.tag_ids) > 0">
+          <!-- thead = table header, the row with column titles -->
+          <thead>
+              <!-- table row element -->
+              <tr>
+                  <!-- table header element -->
+                  <th>ID</th>
+                  <th>Name</th>
+              </tr>
+          </thead>
+          <!-- table body, the main content -->
+          <tbody>
+              <!-- we create a row for each subrecord with t-foreach -->
+              <tr t-foreach="doc.tag_ids" t-as="tag">
+                  <!-- for each line, we output the name and price as table cells -->
+                  <td t-out="tag.id"/>
+                  <td t-out="tag.name"/>
+              </tr>
+          </tbody>
+      </table>
+
+   If you want to display another block in case the `t-if` statement is evaluated as `False`, you
+   can specify it using the `t-else` statement. The `t-else` block must directly follow the `t-if`
+   block in the document structure. There is no need to specify any condition in the `t-else`
+   attribute. As an example, let's show a quick message explaining that there are no tags on the
+   quotation:
+
+   .. code-block:: xml
+      :emphasize-lines: 22
+
+      <!-- table root element -->
+      <table class="table" t-if="len(doc.tag_ids) > 0">
+          <!-- thead = table header, the row with column titles -->
+          <thead>
+              <!-- table row element -->
+              <tr>
+                  <!-- table header element -->
+                  <th>ID</th>
+                  <th>Name</th>
+              </tr>
+          </thead>
+          <!-- table body, the main content -->
+          <tbody>
+              <!-- we create a row for each subrecord with t-foreach -->
+              <tr t-foreach="doc.tag_ids" t-as="tag">
+                  <!-- for each line, we output the name and price as table cells -->
+                  <td t-out="tag.id"/>
+                  <td t-out="tag.name"/>
+              </tr>
+          </tbody>
+      </table>
+      <div class="text-muted" t-else="">No tag present on this document.</div>
+
+   By using the `t-if/t-else` notation, the report editor recognizes that these sections are
+   mutually exclusive and should be displayed as conditional blocks:
+
+   .. image:: pdf_reports/XML-condition-if.png
+      :alt: Output preview if there are tags.
+
+   You can switch conditions using the editor to preview their output:
+
+   .. image:: pdf_reports/XML-condition-else.png
+      :alt: Output preview if there are no tags.
+
+   If you wish to have multiple options, you can also use `t-elif` directives to add intermediary
```

Comment:
I’m not sure if calling these “options” is a good choice.

I would suggest: "You can add more conditions by adding intermediary conditions" (even if it sounds a bit silly).

"Options" implies, at least to me, that you choose just one/switch between several, whether this seems more like multi-level conditions (they all work together and it's not possible to opt out of any of them).

I hope this makes sense ;)

Reply:
Let's discuss this